client/transactions/uncaptured/index.tsx

```diff
@@ -0,0 +1,243 @@
+import { Button } from '@wordpress/components';
+import { onQueryChange, getQuery } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import { useAuthorizations, useAuthorizationsSummary } from 'data/index';
+import Page from '../../components/page';
+import autocompleter from 'transactions/autocompleter';
+
+interface Column extends TableCardColumn {
+	key:
+		| 'authorization_id'
+		| 'authorized_on'
+		| 'captured_by'
+		| 'order'
+		| 'risk_level'
+		| 'amount'
+		| 'customer_email'
+		| 'customer_country';
+	visible?: boolean;
+	cellClassName?: string;
+}
+
+const getColumns = (): Column[] =>
+	[
+		{
+			key: 'authorization_id',
+			label: __( 'Authorization Id', 'woocommerce-payments' ),
+			visible: false,
+			isLeftAligned: true,
+		},
+		{
+			key: 'authorized_on',
+			label: __( 'Authorized on', 'woocommerce-payments' ),
+			screenReaderLabel: __( 'Authorized on', 'woocommerce-payments' ),
+			required: true,
+			isLeftAligned: true,
+			defaultOrder: 'desc',
+			cellClassName: 'date-time',
+			isSortable: true,
+			defaultSort: true,
+		},
+		{
+			key: 'captured_by',
+			label: __( 'Captured by', 'woocommerce-payments' ),
+			screenReaderLabel: __( 'Captured by', 'woocommerce-payments' ),
+			required: true,
+			isLeftAligned: true,
+			defaultOrder: 'desc',
+			cellClassName: 'date-time',
+			isSortable: true,
+			defaultSort: true,
+		},
+		{
+			key: 'order',
+			label: __( 'Order', 'woocommerce-payments' ),
+			screenReaderLabel: __( 'Order number', 'woocommerce-payments' ),
+			required: true,
+		},
+		{
+			key: 'risk_level',
+			label: __( 'Risk level', 'woocommerce-payments' ),
+			screenReaderLabel: __(
+				'Risk level of transaction',
+				'woocommerce-payments'
+			),
+			isLeftAligned: true,
+		},
+		{
+			key: 'amount',
+			label: __( 'Amount', 'woocommerce-payments' ),
+			screenReaderLabel: __( 'Amount', 'woocommerce-payments' ),
+			isNumeric: true,
+			isSortable: true,
+		},
+		{
+			key: 'customer_email',
+			label: __( 'Email', 'woocommerce-payments' ),
+			screenReaderLabel: __( 'Email', 'woocommerce-payments' ),
+			visible: false,
+			isLeftAligned: true,
+		},
+		{
+			key: 'customer_country',
+			label: __( 'Country', 'woocommerce-payments' ),
+			screenReaderLabel: __( 'Country', 'woocommerce-payments' ),
+			visible: false,
+			isLeftAligned: true,
+		},
+	].filter( Boolean ) as Column[]; // We explicitly define the type because TypeScript can't infer the type post-filtering.
+
+const getFormatedAmountFromString = ( string: string ) => {
+	return `${ string.substring( 0, string.length - 2 ) }.${ string.substring(
+		string.length - 2
+	) }$`;
+};
+
+export const AuthorizationsList = (): JSX.Element => {
+	const columnsToDisplay = getColumns();
+	const {
+		authorizationsSummary,
+		isLoading: isSummaryLoading,
+	} = useAuthorizationsSummary( getQuery() );
+
+	const { authorizations, isLoading } = useAuthorizations( getQuery() );
+
+	const getRiskColor = ( risk: string ) => {
```

Comment:
I think you can use the new `RiskLevel` TS type here. To be able to do that you'd need to import the type definition. That makes me thing that maybe it would be a good idea to move the type definition to a dedicated authorizations file under `client/types`, what do you think?

Reply:
I think that makes sense. I moved that to the `client/types`. Now I wonder if we should move all definitions there 🤔

Reply:
Yes, I think it makes sense to move also the definitions in `client/data/authorizations/hooks.ts`.